scripts/vtk_convert.py

```diff
@@ -0,0 +1,91 @@
+from vtk import vtkPolyDataReader, vtkStructuredPointsReader, \
+    vtkXMLImageDataWriter, vtkXMLPolyDataWriter
+from girder_client import GirderClient
+import os
+import click
+import re
+
+@click.command()
+@click.option('--inputfolderid', required=True, prompt=True, help='Source Folder ID')
```

Comment:
Most CLIs use kebab-case naming for multiple words. So, the option should be `--input-folder-id` in this case.